preliminaries.tex

```diff
@@ -1664,7 +1664,7 @@ \subsection{Path induction}
 
 Given a proof $p : \id{a}{b}$,
 path induction requires us to replace \emph{both} $a$ and $b$ with the same unknown element $x$; thus in order to define an element of a family
-$C$, for all pairs of elements of $A$, it suffices to define it on the diagonal.
+$C$, for all pairs of equal elements of $A$, it suffices to define it on the diagonal.
```

Comment:
This is a nice correction. I wonder if we need to be even more careful and explain that "for all pairs of equal elements" means "triples (a,b,p) where p : a = b"?